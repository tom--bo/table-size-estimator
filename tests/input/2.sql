CREATE table `t3` (
  `id` int not null auto_increment,
  `c1` int not null,
  `c2` char(32),
  `c3` varchar(120),
  primary key (`id`),
  unique key `idx1` (`c1`),
  key `idxc13` (`c1`, `c2`)
);
