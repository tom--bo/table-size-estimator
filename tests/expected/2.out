Input Table Definition: (Please type ^d to end input) 
successfully ended
------
1 row max size = 780 bytes .
1 row Average size = 539 bytes .
